requirements/test-requirements.txt

```diff
@@ -2,7 +2,7 @@
 # Required for testing utilities for parsing pip requirements
 pip>=20.1
 ## Test-only dependencies
-pytest==8.1.0
+pytest==8.1.1
```

Comment:
Do we need to pin `pytest`?

Reply:
I think we should to avoid breaking changes in pytest. For example, pytest 9.0 might be released today and break our tests.